### ISSUE TITLE
Invokedynamic: Error: Not dummy Operand from SootMethod.getBody()

### ISSUE THREAD

**Issue by slimming-fat** [soot-oss/SootUp#574]
Error retrieving method body
![image](https://user-images.githubusercontent.com/126549527/229510966-2e66166f-1474-404f-b931-e68a9c31600a.png)

**Comment by jpstotz** [comment]
@Jacky-code519 It would be really nice of you not posting stack traces as image. IntelliJ and Eclipse have such a nice function to parse stacktraces and make them clickable to the code references, but with images this is impossible.

**Comment by JonasKlauke** [comment]
Yeah some additonal information would be nice.
Are you using the released version or the current dev snapshot?

can you send the bytecode which caused this issue?
I can only guess that the InvokeDynamicInstruction in the method causes a problem.

**Comment by slimming-fat** [comment]
@jpstotz @JonasKlauke 
> Yeah some additonal information would be nice. Are you using the released version or the current dev snapshot?
> 
> can you send the bytecode which caused this issue? I can only guess that the InvokeDynamicInstruction in the method causes a problem.

I'm using the released version
Interllij's debug results in the following
![image](https://user-images.githubusercontent.com/126549527/229674910-684e303e-fa49-468e-93d9-f6775bffd87c.png)

The bytecode is as follows：
```java
public static skipUntilByteCount(Lorg/reactivestreams/Publisher;J)Lreactor/core/publisher/Flux;
   L0
    LINENUMBER 450 L0
    ALOAD 0
    LDC "Publisher must not be null"
    INVOKESTATIC org/springframework/util/Assert.notNull (Ljava/lang/Object;Ljava/lang/String;)V
   L1
    LINENUMBER 451 L1
    LLOAD 1
    LCONST_0
    LCMP
    IFLT L2
    ICONST_1
    GOTO L3
   L2
   FRAME SAME
    ICONST_0
   L3
   FRAME SAME1 I
    LDC "'maxByteCount' must be a positive number"
    INVOKESTATIC org/springframework/util/Assert.isTrue (ZLjava/lang/String;)V
   L4
    LINENUMBER 453 L4
    LLOAD 1
    ALOAD 0
    INVOKEDYNAMIC get(JLorg/reactivestreams/Publisher;)Ljava/util/function/Supplier;
[
      // handle kind 0x6 : INVOKESTATIC
      java/lang/invoke/LambdaMetafactory.metafactory(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/invoke/MethodType;Ljava/lang/invoke/MethodType;Ljava/lang/invoke/MethodHandle;Ljava/lang/invoke/MethodType;)Ljava/lang/invoke/CallSite;
      // arguments:
      ()Ljava/lang/Object;, 
      // handle kind 0x6 : INVOKESTATIC
      org/springframework/core/io/buffer/DataBufferUtils.lambda$skipUntilByteCount$16(JLorg/reactivestreams/Publisher;)Lorg/reactivestreams/Publisher;, 
      ()Lorg/reactivestreams/Publisher;
    ]
    INVOKESTATIC reactor/core/publisher/Flux.defer (Ljava/util/function/Supplier;)Lreactor/core/publisher/Flux;
    LDC Lorg/springframework/core/io/buffer/PooledDataBuffer;.class
    INVOKEDYNAMIC accept()Ljava/util/function/Consumer;
[
      // handle kind 0x6 : INVOKESTATIC
      java/lang/invoke/LambdaMetafactory.metafactory(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/invoke/MethodType;Ljava/lang/invoke/MethodType;Ljava/lang/invoke/MethodHandle;Ljava/lang/invoke/MethodType;)Ljava/lang/invoke/CallSite;
      // arguments:
      (Ljava/lang/Object;)V, 
      // handle kind 0x6 : INVOKESTATIC
      org/springframework/core/io/buffer/DataBufferUtils.release(Lorg/springframework/core/io/buffer/DataBuffer;)Z, 
      (Lorg/springframework/core/io/buffer/PooledDataBuffer;)V
    ]
   L5
    LINENUMBER 472 L5
    INVOKEVIRTUAL reactor/core/publisher/Flux.doOnDiscard (Ljava/lang/Class;Ljava/util/function/Consumer;)Lreactor/core/publisher/Flux;
   L6
    LINENUMBER 453 L6
    ARETURN
   L7
    LOCALVARIABLE publisher Lorg/reactivestreams/Publisher; L0 L7 0
    // signature Lorg/reactivestreams/Publisher<+Lorg/springframework/core/io/buffer/DataBuffer;>;
    // declaration: publisher extends org.reactivestreams.Publisher<? extends org.springframework.core.io.buffer.DataBuffer>
    LOCALVARIABLE maxByteCount J L0 L7 1
    MAXSTACK = 4
    MAXLOCALS = 3

  // access flags 0x9
  // signature <T::Lorg/springframework/core/io/buffer/DataBuffer;>(TT;)TT;
  // declaration: T retain<T extends org.springframework.core.io.buffer.DataBuffer>(T)
```

**Comment by SoniaZaldana** [comment]
are there any updates on this issue? I'm also seeing failures with invoke dynamic instructions in resolving method bodies.

**Comment by swissiety** [comment]
possibly the same underlying error as #666

**Comment by swissiety** [comment]
#666 is merged - @slimming-fat does the error still occur with the current develop branch as dependency?

**Comment by swissiety** [comment]
stale - probably fixed by #666
